Arduino_package/hardware/libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -66,11 +66,6 @@ WiFiClient::WiFiClient(uint8_t sock, tProtMode portMode, tBlockingMode blockMode
     _is_blocked = blockMode;
 }
 
-WiFiClient::~WiFiClient()
-{
-    stop();
```

Comment:
Is there any reason that remove this `stop` function?

Reply:
How about the following update 

```
if (_is_blocked) {
    stop();
}
```

Reply:
@redelacruz   I think the suggested fix is ok. How about you?